mobile/native/src/calculations/mod.rs

```diff
@@ -25,9 +25,12 @@ pub fn calculate_pnl(
     leverage: f32,
     direction: Direction,
 ) -> Result<i64> {
+    // FIXME: We can no longer assume that the coordinator always has the same leverage! It needs to
```

Comment:
I noticed that this was wrong so I changed it to the most correct value that we can possibly hard-code, but we need to find a way to know the current coordinator liquidity for a channel on the app so that we can correctly cap the profits of the trader.

I don't think we should bother fixing this now, but I will open an issue (later).